lib/services/validator_service/validator_service.dart

```diff
@@ -1,13 +1,23 @@
 import 'package:provenance_dart/wallet.dart';
+import 'package:provenance_wallet/screens/home/explorer/explorer_bloc.dart';
 import 'package:provenance_wallet/services/models/provenance_validator.dart';
+import 'package:provenance_wallet/services/models/validator_delegate.dart';
 import 'package:provenance_wallet/util/strings.dart';
 
 abstract class ValidatorService {
   Future<List<ProvenanceValidator>> getRecentValidators(
+    Coin coin,
+    int pageNumber,
+    ValidatorStatus status,
+  ) {
+    throw Strings.notImplementedMessage;
+  }
+
+  Future<List<ValidatorDelegate>> getDelegations(
```

Comment:
What about renaming `ValidatorDelegate` to `Delegation`, since "delegate" might typically refer to some type of class delegate.

Reply:
I can see that, I'll make those changes.